map/benchmark_tool/benchmark_tool.pro

```diff
@@ -6,6 +6,7 @@ CONFIG -= app_bundle
 TEMPLATE = app
 
 ROOT_DIR = ../..
+# TODO(AlexZ): Remove map dependency.
```

Comment:
To remove dependency, we need to change it’s code. And move only after that. It’s for separate PR and I put TODO for it.